cms/templates/cms/program_page.html

```diff
@@ -6,6 +6,20 @@
 
 {% block title %}{% trans "MIT Micromasters" %}{% endblock %}
 
+{% block social %}
+  <meta property="og:site:name" content="{{ page.title }} MicroMasters">
+  <meta property="og:type" content="website">
+  <meta property="og:url" content="{{ program.programpage.url }}">
+  <meta property="og:title" content="{{ page.title }} MicroMasters">
+  <meta property="og:description" content="{% if page.program.description %}{{ page.program.description }}{% endif %}">
+  <meta property="og:image" content="{{ page.background_image }}">
```

Comment:
I put a program page in the PR build and it looks like this is just the filename of the image. Can you figure out how to get the absolute URL here?